types/lodash/common/lang.d.ts

```diff
@@ -572,10 +572,10 @@ declare module "../index" {
          * @return Returns true if value is empty, else false.
          */
         isEmpty<T extends { __trapAny: any }>(value?: T): boolean;
-        isEmpty(value: string): value is '';
-        isEmpty(value: Map<any, any> | Set<any> | List<any> | null | undefined): boolean;
```

Comment:
I feel uneasy changing this part. Is it required to keep listing `| null | undefined` for other overrides?